### DISCUSSION TITLE
Can I display controls and view on separate screens?

### DISCUSSION THREAD

**Discussion by DJamesTePapa** [Stellarium/stellarium#2030] [Q&A]
I have recently updated Strellarium from a very old version and discovered that the controls are now always visible over the view of the sky. I use the software with a curved mirror to project onto an inflatable dome and I don't want to see the controls projected as they distract from the view.
Is it possible to separate out the controls and view of the sky onto different screens?

**Comment by alex-w** [comment]
Please use Remote Control plugin for it

**Reply by gzotti** [reply]
Yes, the RemoteControl was made for this. And earlier, the Text User Interface. Both would need a bit attention, though, as the latest options have not yet been integrated. 
Which was the very old version that allowed to have controls on another screen?

**Comment by alphanimal** [comment]
I was thinking the same earlier today - it would be useful to have the options etc. in a separate window some times

**Reply by gzotti** [reply]
Yes, the RemoteControl was made for this purpose.
